python/cudf/cudf/core/dataframe.py

```diff
@@ -5426,6 +5436,9 @@ def _prepare_for_rowwise_op(self, method, skipna):
             mask = None
 
         coerced = filtered.astype(common_dtype)
```

Comment:
Similarly for this as well?
```suggestion
        coerced = filtered.astype(common_dtype, copy=False)
```

Reply:
Yup! Good catch